draft-smyslov-ipsecme-ikev2-reliable-transport.xml

```diff
@@ -155,84 +227,64 @@ N(SEPARATE_TRANSPORTS)  --->
                                    [N(NAT_DETECTION_SOURCE_IP),
                                    N(NAT_DETECTION_DESTINATION_IP),]
                              <---  N(SEPARATE_TRANSPORTS)
-          ]]></artwork></figure>
-
-          <t>If request was sent over UDP, then the initiator <bcp14>MUST</bcp14> switch to TCP using destination port
-          4500 in the next exchange (IKE_INTERMEDIATE or IKE_AUTH) and the
-          responder <bcp14>MUST</bcp14> be prepared to receive the next exchange request message
-          on TCP port 4500.
-          </t>
-
-          <figure><artwork align="center" ><![CDATA[
-Initiator (TCP)                    Responder (TCP:4500)
--------------------------------------------------------------------
-HDR, SK{...}  --->
-                             <---  HDR, SK{...}
-          ]]></artwork></figure>
```

Comment:
I think that this figure should not go away. Perhaps the right place for it - just below the first figure, depicting feature negotiation in IKE_SA_INIT.